File: Kdn/JS/Kdn/editor/UserGroup.js
Kdn.editor.UserGroup = Ext.extend(Ext.net.MultiCombo, {
    initComponent:function(){
        Ext.apply(this, {
            displayField: 'UserGroupName',
            valueField: 'UserGroupId',
            triggerAction: 'all',
            mode: 'local',
            store:Kdn.ModelFactory.getStore('UserGroup')
        });
        Kdn.editor.UserGroup.superclass.initComponent.call(this);
    }
});
Ext.reg('kdn.editor.usergroup', Kdn.editor.UserGroup);

File: Transport/OtherModels/farm/UserGroup.cs
using Kdn.Ext.Attributes;

namespace Transport.OtherModels.farm
{

        
        public class UserGroup
        {
            public const string FieldsMapping = @"[
                'UserGroupId',
                'UserGroupName'
            ]";

            [IdProperty]
            public virtual int UserGroupId { get; set; }
            public virtual string UserGroupName { get; set; }
        }
}
